src/main/scala/devices/debug/Debug.scala

```diff
@@ -722,6 +721,263 @@ class TLDebugModuleInner(device: Device, getNComponents: () => Int, beatBytes: I
       }
     }
 
+    //--------------------------------------------------------------
+    // System Bus Access
+    //--------------------------------------------------------------
+    // --- SBCS Registers ---
+
+    sb2tlOpt match {                   
```

Comment:
rather than the generic `match { case (Some)... case (None) }` pattern you could use: `sb2tlOpt.foreach { sb2tl => ...}`.  This is preferable I believe. Or if you want to return something you can use `sb2tlOpt.map{...}.getOrElse{...}` (see below)